specs/DesignDiscussions/InputLabels.md

```diff
@@ -72,17 +72,55 @@ This conflict speaks to a general tension inherent in the design of Adaptive Car
 
 In order to gather more data to help us balance the needs of the card authors and hosts, we will send a survey to customers who have expressed an interest in improving out input story. The following options are under consideration, and will be evaluated against survey data to determine which if any will be included in our v1 implementation.
```

Comment:
> we will send a survey to customers [](start = 88, length = 35)

@ almedina-ms can you update the spec with a summary of what the results of the survey were around this - Summary: 
At this point looks like app developers want that control in hostconfig to match label formatting that they have for their respective apps; while the majority of the card authors are also used to having flexibility around being able to format the default label text.